### ISSUE TITLE
Most of the addin fonctionnalities are not available

### ISSUE THREAD

**Issue by Nours21** [rubberduck-vba/Rubberduck#6226]
Hello,
most of the fonctionnality of the addin are missing so this must have something to do with my setup...
any help on the error recorded in following log would be deeply appreciated!

**Rubberduck version information**
these are the data from the rubberduck about menu
```
Version 2.5.9.6316
OS: Microsoft Windows NT 10.0.22631.0, x64
Host Product: Microsoft Office x64
Host Version: 16.0.17830.20210 (O365)
Host Executable: EXCEL.EXE
```
please note that the actual OS is W11

**Description**
after refreshing the parser, most of the Rubberduck menu are still greyed out, especially the "refractor" menu
the "rename" option can still be accessed by its keyboard shortcut
       ![image](https://github.com/user-attachments/assets/e2da6f1c-cf41-4323-b341-e4d644fb208e)

the rubberduck toolbar display only "ready" and 2 blank spaces
      ![image](https://github.com/user-attachments/assets/8150bee8-61e9-458b-a6b7-7a5943d641f1)
the "code inspections" window work as expected

the log file include following error
```
2024-10-04 16:05:27.4330;ERROR-2.5.9.6316;Rubberduck.Parsing.VBA.RubberduckParserState;An exception occurred when notifying consumers of updated parser state.;System.InvalidCastException: Impossible d'effectuer un cast d'un objet COM de type 'System.__ComObject' en type d'interface 'Microsoft.Vbe.Interop.VBE'. Cette opération a échoué, car l'appel QueryInterface sur le composant COM pour l'interface avec l'IID '{0002E166-0000-0000-C000-000000000046}' a échoué en raison de l'erreur suivante : Erreur lors du chargement de la bibliothèque/DLL du type. (Exception de HRESULT : 0x80029C4A (TYPE_E_CANTLOADLIBRARY)).
   à System.StubHelpers.StubHelpers.GetCOMIPFromRCW(Object objSrc, IntPtr pCPCMD, IntPtr& ppTarget, Boolean& pfNeedsRelease)
   à Microsoft.Vbe.Interop.VBE.get_VBProjects()
   à Rubberduck.VBEditor.SafeComWrappers.VBA.VBE.get_VBProjects()
   à Rubberduck.VBEditor.SafeComWrappers.VBA.VBE.get_IsInDesignMode()
   à Rubberduck.VBEditor.ComManagement.NonDisposalDecorators.VbeNonDisposalDecorator`1.get_IsInDesignMode()
   à Rubberduck.UnitTesting.TestEngine.StateChangedHandler(Object sender, ParserStateEventArgs e)
   à System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   à Rubberduck.Parsing.VBA.RubberduckParserState.OnStateChanged(Object requestor, CancellationToken token, ParserState state, ParserState oldStatus)
```

**To Reproduce**
Fresh Rubberduck install on my machine (per user)
Refresh the parser

**Expected behavior**
full addin fonctionnalities available

**Screenshots**
included in descritption

**Logfile**
[RubberduckLog.txt](https://github.com/user-attachments/files/17260073/RubberduckLog.txt)

**Additional context**
i do not have admin right on my machine.

**Comment by retailcoder** [comment]
Hi, thanks for the feedback! Would you happen to have any other VBE addins loaded? The InvalidCastException involving the VBE interface is known to happen in such cases, or in cases where the Office COM registration is broken, for example if Office components were installed out of order (e.g. installing Visio 2013 after installing Office 2016).

Try disabling other VBE admins if any, or repairing Office if your machine has had multiple versions installed or running side by side - might need to uninstall Office and reinstall things in release order.

Basically what's happening is that Rubberduck is trying to get ahold of the COM objects for the Office CommandBar buttons and menus, and then it's not able to do that for some reason that's tied to your local environment.

**Comment by Nours21** [comment]
thank you for your help, I will try to get my office suite reinstalled 
on my previous OS I may have been in this case (visio 2010 installed after office 2016) but since i've been migrated to w11/Office 365 every office component should be from the same O365 version

I removed the other installed vbe addin i had without much success

I close this thread as my problems are not linked to the addin itself

thank you for your help!
